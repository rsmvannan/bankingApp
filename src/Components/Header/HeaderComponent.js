import React from "react";
import "./Header.css";
import { Navbar, Nav } from "react-bootstrap";
// import { Link } from "react-router-dom";
// import Authentication from "./Authentication";

const HeaderComponent = (props) => {
  return (
    <Navbar className="header" bg="dark" variant="dark">
      Banking Management System
    </Navbar>
  );
};

export default HeaderComponent;
