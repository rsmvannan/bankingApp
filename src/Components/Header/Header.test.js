import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import HeaderComponent from "./HeaderComponent";

configure({ adapter: new Adapter() });

describe("Header Component should be rendered", () => {
  it("Should be rendered", () => {
    const component = shallow(<HeaderComponent />);
    const wrapper = component.find(".header");
    expect(wrapper.length).toBe(1);
  });
});
