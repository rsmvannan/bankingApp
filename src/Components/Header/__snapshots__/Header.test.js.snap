// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should be rendered Should be rendered 1`] = `ShallowWrapper {}`;

exports[`Header Component should be rendered renders properly 1`] = `ShallowWrapper {}`;
